website/static/js/components/institution.js

```diff
@@ -41,6 +41,31 @@ var InstitutionImg = {
     }
 };
 
+var CheckableInst = {
```

Comment:
Minor: For naming consistency, don't abbreviate `Institution` -> `CheckableInstitution`.